src/ClassApplication/Services/DBService.cs

```diff
@@ -0,0 +1,112 @@
+using ACMTTU.NoteSharing.Shared.SDK.Services;
+using Microsoft.Azure.Cosmos;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using System;
+using ACMTTU.NoteSharing.Shared.SDK.Controllers;
+using ACMTTU.NoteSharing.Platform.NotesApplication.Services;
+using NotesApplication;
+
+namespace ACMTTU.NoteSharing.Platform.NotesApplication.Services
+{
+
+    /// <summary>
+    /// An abstract class for database services
+    /// </summary>
+    public interface IDBService
+    {
+
+        Task<T> CreateItem<T>(T undatabasedObject);
+        Task<T> ReadItem<T>(string id);
+        Task<ItemResponse<T>> ReplaceItem<T>(T update, string id);
+        Task<ItemResponse<T>> DeleteItem<T>(string id);
+        Task<FeedIterator<T>> GetItemQueryIterator<T>(QueryDefinition query);
+
+    }
+
+    /// <summary>
+    /// An implementation of IDBService to interface with Azure Cosmos DB
+    /// </summary>
+    public class DBService : PlatformBaseService, IDBService
+    {
+        private Container _container;
+
+        public DBService(IHttpClientFactory clientFactory) : base(clientFactory) { }
+
+        public async override Task Setup()
+        {
+            var client = await this.dbClient.CreateDatabaseIfNotExistsAsync("NoteDatabase");
```

Comment:
See if you can set this up for the ClassApp. You might have to go into the Azure Portal to get the right name for the Container